bundle.Dockerfile

```diff
@@ -15,19 +15,6 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
-# Red Hat labels.
-LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.openshift.versions="v4.7"
-LABEL \
-    com.redhat.component="elasticsearch-operator" \
-    version="v1.1" \
-    name="elasticsearch-operator" \
-    License="Apache-2.0" \
-    io.k8s.display-name="elasticsearch-operator bundle" \
-    io.k8s.description="bundle for the elasticsearch-operator" \
-    summary="This is the bundle for the elasticsearch-operator" \
-    maintainer="AOS Logging <aos-logging@redhat.com>"
-
```

Comment:
Please revert this change. This is an unfortunate hindsight of `make bundle` still.